src/main/webapp/style.css

```diff
@@ -11,10 +11,25 @@ a:hover,
 
 .btn {
   background-color: rebeccapurple;
+  border: none;
   border-radius: 8px;
+  color: white;
+}
+
+.btn:hover {
+  background-color: purple;
   border: none;
+  border-radius: 8px;
   color: white;
-  width: 70%;
+}
+
+.btn.disabled {
+  background-color: rebeccapurple;
+  color: white;
+}
+
+.btn-shadow:hover {
```

Comment:
Q: To clarify: now Obi and I need to add this class if we want box shadows on hover? (if we need to make this change plz throw it in 3 peas chat when u merge so he and I can change our code accordingly)

Reply:
Yeah you will. Sorry for doing this but it'll be nice to have the option of a shadow on some buttons. I'll send out a chat now.